src/components/HeaderProfile/HeaderProfile.scss

```diff
@@ -0,0 +1,64 @@
+@use "../../styles/variables" as *;
+
+* {
```

Comment:
Can you get rid of this * part, it's done in main.scss, which is imported to App.jsx, so it will be everywhere